examples/machine_vision_object_pickup/main.py

```diff
@@ -0,0 +1,89 @@
+# Automata Technologies - Andrea Antonello, 2019
+import socket
+from automata import Eva
+from evaUtilities import *
+
+# Connection to robot
+host = <user_input>
+token = <user_input>
+eva = Eva(host, token)
+eva.lock_status()
+
+# Connection to camera
+sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+server = <user_input>
+port = <user_input>
+sock.connect((server, port))
+
+# Use-case parameters
+objects = ['C', 'M', 'R']
+object_heights = {'C': <user_input>, 'M': <user_input>, 'R': <user_input>}  # object thicknesses [m]
+end_effector_length = <user_input>  # length of tool [m]
+hover_height = <user_input>  # elevation of idle z axis wrt to the object [m]
+surface_height = <user_input>  # elevation of the pickup surface wrt to the robot [m, signed]
+joint_cal_zero = <user_input>  # joints @ (0,0) of calibration board
+joint_guess = <user_input>  # joints guess for pickup/hover position
+
+with eva.lock():
+    while True:
+        passed, cam_string = readTcpIp(sock) # String format = ['start', 'pattern', 'x_mm', 'y_mm', 'angle', 'pass']:
```

Comment:
There is some case issues between the two files, `readTcpIp` doesn't exist, but `_read_tcp_ip` does.
Overall, try to stick to `snake_case` as this is the standard in Python.